setup.py

```diff
@@ -25,11 +29,10 @@
         "spacy",
         "tqdm",
         "transformers",
-        "ujson"
-    ], 
-    extras_require={
-        'faiss-gpu': ['faiss-gpu>=1.7.0'],
-        'torch': ['torch==1.13.1']
-    },
+        "ujson",
+        "faiss-cpu",
```

Comment:
Oh why make faiss-cpu required

Reply:
I'm fine to take it off, but I felt like having the CPU version installed as part of the main/PyPi package is good for accessibility -- means it's as close to batteries-included and can be installed on any system (and one can still install the gpu version if they want GPU indexing)

Reply:
I guess the issue is that if someone has faiss from conda, which is the recommended way, this will break it?

Reply:
It should probably be colbert-ir[faiss-cpu] ?

Reply:
> It should probably be colbert-ir[faiss-cpu] ?

I think that can work yeah! I was considering this, but then it felt perhaps a bit strange that installing the "main" package through PyPi wouldn't work out-of-the-box... though this wouldn't be the first lib to require an extra dependency

Reply:
> I guess the issue is that if someone has faiss from conda, which is the recommended way, this will break it?

Fair point, I'm not 100% sure but I think you might be right